src/components/Core/Player/Player.jsx

```diff
@@ -53,7 +63,80 @@ class Player extends React.Component {
   }
 
   handleNext() {
-    this.props.mk.instance.player.skipToNextItem();
+    const song = this.props.mk.instance.player
+    song.skipToNextItem();
+
+    if (song.repeatMode === 1) {
+      song.seekToTime(0);
+    }
+  }
+
+  handleAddToLibrary() {
+    const music = this.props.mk.instance;
+    music.player.addToLibrary = 1;
+  }
+
+  handleRepeat() {
+    const music = this.props.mk.instance;
+    const {isRepeatOn} = this.state;
+
+    if (music.player.repeatMode === 0) {
+      this.setState({
+        isRepeatOn: true,
```

Comment:
State is useless in this case, it is already stored in `music.player`

Reply:
I use it for adding the classes tho.